modules/chamber/Makefile

```diff
@@ -2,8 +2,28 @@ CURL ?= curl
 CHAMBER_VERSION ?= 2.0.0
 CHAMBER ?= chamber
 
+ifdef AWS_VAULT
+	CHAMBER_AUTH = ""
+else
+	CHAMBER_AUTH = $(WITH_AWS)
+endif
+
+WITH_CHAMBER = $(CHAMBER_AUTH) chamber exec $(CHAMBER_SERVICES) --
+CHAMBER_EXPORT = $(CHAMBER_AUTH) chamber export -f dotenv $(CHAMBER_SERVICES) | xargs -I{} echo " {}" | tr '\n' ' '
+
+INJECT_CHAMBER ?= $(foreach var, IN_CHAMBER_MODE=true $(shell $(CHAMBER_EXPORT)), $(eval export $(var)))
```

Comment:
This part has me concerned. I want to avoid `eval` *at all cost*; developers just don't get it that `Makefile` is a template.

Reply:
Also, I don't see where `INJECT_CHAMBER` is used

Reply:
https://github.com/sportech-dev/local-dev/blob/chamber-support/Makefile#L14

Reply:
The idea:
Add following line in place you would like to have secrets from chamber:
```
$(INJECT_CHAMBER)
```

That line will export secrets as environment variables 
Where 
* `CHAMBER_EXPORT` export secrets in `.env` format join it into one line separated by space 
* `INJECT_CHAMBER` for each secret (+ IN_CHAMBER_MODE) run `export`

Reply:
I just don't get why we we would ever want to do this rather than use `chamber` the way it was destined.

Reply:
There are 2 ways to use chamber with make.
1) `make chamber/shell` and then run any make target with secrets.
2) Inject secrets
The minus of the first approach is that outside of `chamber/shell` make targets would not work, because they use secret environment variables. 
Ex.: `make compose/up`

The second solution save us from that problem, it export secrets for any make run.
But it cost additional time to request secrets.

I'd prefer to have ability to use both solutions.

Reply:
I am still not convinced we want/need this. Show me next sync up.

Reply:
(I went down this *dangerous* path with Gladly. Our `Makefiles` were only understood by me, and me alone. It does not payoff to be clever with make.